Tests/SwiftStdlibTests/DictionaryExtensionsTests.swift

```diff
@@ -163,4 +163,29 @@ final class DictionaryExtensionsTests: XCTestCase {
             [6: ["Bryant"], 5: ["James"], 4: ["Wade", "John"], 0: ["", ""]])
         XCTAssertEqual(Dictionary(grouping: array3, by: \String.count), [:])
     }
+
+    func testGetByKeys() {
+        let dict = ["James": 100,
+                    "Wade": 200,
+                    "Bryant": 500,
+                    "John": 600,
+                    "Jack": 1000]
+        let picked = dict.pick(keys: ["James", "Wade", "Jack"])
+        XCTAssertEqual(picked, ["James": 100, "Wade": 200, "Jack": 1000])
+        let noResults = dict.pick(keys: ["Michael", "Joe", "Christian"])
+        XCTAssertTrue(noResults.isEmpty)
+
+        let optionalValuesDict = ["James": 100,
+                                  "Wade": nil,
+                                  "Bryant": 500,
+                                  "John": nil,
+                                  "Jack": 1000]
+
+        let pickedWithOptioanls = optionalValuesDict.pick(keys: ["James", "Bryant", "John"])
```

Comment:
Can you just add an additional test with a key that doesn't exist in the original dictionary?